docs/running-clusterfuzzlite/prow.md

```diff
@@ -0,0 +1,238 @@
+`<your-cloud-bucket>` in the config examples in this document. If
+The bucket will be used to store crashes, builds, corpora, and coverage reports.
+
+Then you must create a service account with access to both `<your-cloud-bucket>` and the Prow Results Bucket
+To do this
+  1) Create a Cloud Service account in your Project
+
+    gcloud beta iam service-accounts create `<SA_NAME>` --project=`<PROJECT_ID>` --description=`<SA_DESCRIPTION>` --display-name=`<SA_DISPLAY_NAME>`
+
+  2) Give the Cloud Service account access to BOTH the k8s Prow Results Bucket AND `<your-cloud-bucket>` using the command
+
+    gsutil iam ch "serviceAccount:${SA}:roles/storage.objectAdmin" "${BUCKET}"
+    
+  3) Create a new K8s service account with a distinct name (called `<K8s_SA>` for the rest of this documentation)
+
+  4) Add the Google Cloud Service account to the annotations for the K8s service account like so:
+      ```
+      apiVersion: v1
+      kind: ServiceAccount
+      metadata:
+        annotations:
+          iam.gke.io/gcp-service-account: <YOUR_CLOUD_SERVICE_ACCOUNT>
+        name: `<K8s_SA>`
+        namespace: test-pods
+      ```
+  5) Run [bind-service-accounts.sh](https://github.com/kubernetes/test-infra/tree/f7e21a3c18f4f4bbc7ee170675ed53e4544a0632/workload-identity) to bind `<K8s_SA>` to your Google Cloud Service Account
+
+TODO(mpherman): Create a script to automatically ensure Service Accounts and Buckets to reduce setup steps.
+
+Note that the configuration examples set the environment variable
+`CFL_PLATFORM` these values are general for all GCB users. Do not
+change them.
+
+## Presubmit/Postsubmit fuzzing
+
+To add a fuzzing workflow that runs on all pull requests to your project, add
+the following to your job config under either Presubmits or Postsubmits:
+
+{% raw %}
+```yaml
+- name: <YOUR-PROWJOB-NAME>
+  labels:
+    preset-dind-enabled: "true"
+  decorate: true
+  spec:
+    serviceAccountName: <K8s-SA>
+    containers:
+    - image: gcr.io/k8s-testimages/ci_fuzz:<VERSION>
+      command:
+        - runner.sh
+      args:
+        - python3
+        - "/opt/oss-fuzz/infra/cifuzz/cifuzz_combined_entrypoint.py"
+      # docker-in-docker needs privileged mode
+      securityContext:
+        privileged: true
+      env:
+      - name: MODE
+        value: code-change
+      - name: SANITIZER
+        value: <SANITIZER>
+      - name: CLOUD_BUCKET
+        value: <YOUR-CLOUD-BUCKET>
+      - name: CFL_PLATFORM
+        value: prow
+```
+{% endraw %}
+
+You can set the `FUZZ_SECONDS` env var when running the fuzzers to change the
+amount of time ClusterFuzzLite will fuzz PRs for.
+
+## Periodic Fuzzing
+
+To enable batch fuzzing, add the following to your prow job config for periodic jobs.
+
+{% raw %}
+```yaml
+- name: <YOUR-PROWJOB-NAME>
+  labels:
+    preset-dind-enabled: "true"
+  decorate: true
+  extra_refs:
+  - org: <ORG>
+    repo: <REPO>
+    base_ref: <BRANCH>
+  interval: 1h # THIS CAN BE A CRON AS WELL
```

Comment:
Isn't this going to run hourly? How can we make it run daily?

Reply:
interval: 24h